AMSoftware.Crm.PowerShell.Commands/Administration/GetSPDocLoc.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AMSoftware.Crm.PowerShell.Common;
+using AMSoftware.Crm.PowerShell.Common.ArgumentCompleters;
+using AMSoftware.Crm.PowerShell.Common.Repositories;
+using Microsoft.Crm.Sdk.Messages;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Query;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AMSoftware.Crm.PowerShell.Commands.Administration
+{
+    [Cmdlet(VerbsCommon.Get, "CrmSPDocLoc", HelpUri = HelpUrlConstants.GetRoleHelpUrl, SupportsPaging = true, DefaultParameterSetName = "")]
```

Comment:
I prefer fullnames for my Cmdlets. Autocomplete in PowerShell will help with typing, and this way the comdlets are clear.
Add a HelpUrl in the HelpUrlConstants, or leave out if no help provided.
```suggestion
    [Cmdlet(VerbsCommon.Get, "CrmSPDocumentLocation", HelpUri = HelpUrlConstants.GetSPDocumentLocationHelpUrl, SupportsPaging = true, DefaultParameterSetName = "")]
```

Reply:
accepted